backend/src/domain/models/Game.js

```diff
@@ -78,6 +80,7 @@ export class Game {
    * This emits an end_game event to all users in the session
    */
   async endGame() {
+    // delay to let db sync, thne tell suers game has ended
```

Comment:
I think there are a couple of typos in this comment. Specifically, "thne" and "suers". Would be nice if you could change those to "then" and "users". :)

Reply:
Yea, would also be good if punctuation was consistent. Like sentenses starting with a captical letter and ending with (.).

Reply:
Oo, thank you was in a bit of a rush I can fix that later today